stdio/scanf.c

```diff
@@ -154,8 +154,13 @@ static int scanf_parse(char *ccltab, const char *inp, int *inr, char const *fmt0
 
 		width = 0;
 		flags = 0;
+		int convType = CT_NONE;
```

Comment:
Variable declarations should be placed at the start of the block, at least.

Reply:
Done